flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/AfterMatchSkipStrategy.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.cep.nfa;
+
+import java.io.Serializable;
+
+
+/**
+ * Indicate the skip strategy after a match process.
+ * <p>There're four kinds of strategies:
+ * SKIP_PAST_LAST_EVENT,
+ * SKIP_TO_NEXT_EVENT,
+ * SKIP_TO_FIRST_<code>PATTERN</code> and
+ * SKIP_TO_LAST_<code>PATTERN</code>.
+ * </p>
+ */
+public class AfterMatchSkipStrategy implements Serializable {
+
+	// default strategy
+	SkipStrategy strategy = SkipStrategy.SKIP_TO_NEXT_EVENT;
+
+	// pattern name to skip to
+	String patternName = null;
+
+	/**
+	 * Skip to first *pattern*.
+	 * @param patternName the pattern name to skip to
+	 * @return
```

Comment:
The `@return` javadoc is missing on all of the methods. Also add a brief description of what each one implies.